content/en/docs/faq.md

```diff
@@ -48,7 +48,8 @@ Here's a [video we like](https://www.youtube.com/watch?v=Xe1TZaElTAs) about the
 
 We recommend reporting such sites to Google Safe Browsing and the Microsoft Smart Screen program, which are able to more effectively protect users. Here is the Google reporting URL:
```

Comment:
"Here is the Google reporting URL" should say something like "Here are the reporting URLs"

Reply:
Good catch